testsuite/MDAnalysisTests/__init__.py

```diff
@@ -104,13 +104,13 @@
    http://projects.scipy.org/numpy/wiki/TestingGuidelines#id11
 .. _Charmm: http://www.charmm.org
 .. _Gromacs: http://www.gromacs.org
+
 """
 
 __version__ = "0.11.0-dev"  # keep in sync with RELEASE in setup.py
 
 try:
-    from numpy.testing import Tester, assert_, assert_equal, TestCase
-    test = Tester().test
+    from numpy.testing import assert_
 except ImportError:
     raise ImportError("""numpy>=1.5  is required to run the test suite. Please install it first. """
```

Comment:
Please make sure to update the numpy version requirement in both `setup.py` (package and testsuite) if we don't require 1.5+ already.